test/integration/puppetlabs/services/jruby/tasks_test.clj

```diff
@@ -181,17 +181,9 @@
 
 (deftest ^:integration all-tasks-test
   (testing "requesting all tasks"
-    (let [tasks [{:name "install"
-                  :module-name "apache"
-                  :metadata (simple-impl-metadata "apache")
-                  :number-of-files 2}
-                 {:name "init"
+    (let [tasks [{:name "init"
```

Comment:
Would it still be useful to test with multiple tasks?

Reply:
I don't think it adds anything. (I was actually torn about removing the test) Really we're just making sure the JRuby API exists and returns valid data, the functionality of listing valid tasks is well tested in Puppet (what JRuby is calling into).